### PR TITLE
OSDOCS-2950: Adding support default IPI cluster installation to Azure…

### PR DESCRIPTION
**Pull request by tmalove**
Preview link: https://deploy-preview-42670--osdocs.netlify.app/openshift-enterprise/latest/installing/installing-preparing.html

Applies to: 4.10
Urgency level: High
Based on PR# https://github.com/openshift/openshift-docs/pull/42541/files

### CODE REVIEW

File: modules/installation-obtaining-installer.adoc [hunk 1]
```diff
@@ -88,7 +91,7 @@ endif::ash[]
 . Navigate to the page for your installation type, download the installation program that corresponds with your host operating system and architecture, and place the file in the directory where you will store the installation configuration files.
 endif::[]
 ifdef::openshift-origin[]
-. Download installer from https://github.com/openshift/okd/releases
+. Download installer from https://github.com/openshift/okd/releases.
```
Comment (jboxman) [inline modules/installation-obtaining-installer.adoc:94]: My suspicion is that the period here will be included in the URL.

File: installing/installing_azure_stack_hub/installing-azure-stack-hub-network-customizations.adoc [hunk 2]
```diff
@@ -0,0 +1,85 @@
+[id="installing-azure-stack-hub-network-customizations"]
+= Installing a cluster on Azure Stack Hub with network customizations
+include::modules/common-attributes.adoc[]
```
Comment (jboxman-rh) [inline installing/installing_azure_stack_hub/installing-azure-stack-hub-network-customizations.adoc:3]: So I think this needs to be `_attributes/common-attributes.adoc` to work now.

File: modules/installation-obtaining-installer.adoc (hunk 1 above)
Comment (jboxman-rh) [inline modules/installation-obtaining-installer.adoc:94]: Looks like we don't do OKD previews anymore, but I think adding a period here will break the link.

**Comment by jboxman-rh** [comment]
Bizarrely, the TOC is missing from the preview on the right.

https://deploy-preview-42670--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_azure_stack_hub/installing-azure-stack-hub-network-customizations.html

File: installing/installing_azure_stack_hub/installing-azure-stack-hub-network-customizations.adoc (hunk 2 above)
Comment (tmalove) [reply installing/installing_azure_stack_hub/installing-azure-stack-hub-network-customizations.adoc:3]: Update complete.

File: modules/installation-obtaining-installer.adoc (hunk 1 above)
Comment (tmalove) [reply modules/installation-obtaining-installer.adoc:94]: Update complete.

**Comment by tmalove** [comment]
Update complete (add modules in the Modules included...section of one assembly file.

**Comment by tmalove** [comment]
@jboxman-rh  after I made the updates, I squashed my commits but noticed that the number of my commits in the log was 2 and the number of commits in the PR was 3. I only squashed 2 of the commits, although I saw that the commit after my last one was Mike P's.

I did not include Mike's commit, just from a logistical point of view, but I did notice that the commit number in the log output was the same as Mike's commit number in this PR...see the following:
git-log

@mjpytlak just checking that I was correct in not including your commit after mine from the 'git log' command?

Thanks to your both!

**Comment by mjpytlak** [comment]
Good morning, Tami.

When you squash, you need to account for the number of commits in the PR,
and if necessary, any commits locally that have not been pushed in the PR.
For example, if you have 3 commits in the PR and 1 commit locally, you
would enter the following:

$ git rebase -i HEAD~4.

I hope that helps,
-Mike

**Comment by tmalove** [comment]
Thanks for your reply @mjpytlak. I will verify with @jboxman-rh but I was told, as a best practice, to squash only the commits with the author's name (although the adjacent commit with your name is also a part of my PR from the work that you did). I will let you know how it turns out.

**Comment by jboxman-rh** [comment]
@tmalove, LGTM, thanks!

**Pull request merged**